edgecast/customer/customer.go

```diff
@@ -0,0 +1,368 @@
+// Copyright Edgecast, Licensed under the terms of the Apache 2.0 license.
+// See LICENSE file in project root for terms.
+
+package customer
+
+import (
+	"fmt"
+
+	"github.com/EdgeCast/ec-sdk-go/edgecast/internal/urlutil"
+)
+
+// AddCustomer creates a new Customer under the Partner associated with the API
+// token used for this request.
+func (svc *CustomerService) AddCustomer(
+	params AddCustomerParams,
+) (string, error) {
+	relURL := "v2/pcc/customers"
+	if params.Customer.PartnerUserID != 0 {
+		relURL = relURL + fmt.Sprintf("?partneruserid=%d", params.Customer.PartnerUserID)
+	}
+
+	request, err := svc.Client.BuildRequest("POST", relURL, params.Customer)
+
+	if err != nil {
+		return "", fmt.Errorf("AddCustomer: %v", err)
+	}
+
+	parsedResponse := &struct {
+		AccountNumber string `json:"AccountNumber"`
+	}{}
+
+	_, err = svc.Client.SendRequest(request, &parsedResponse)
+
+	if err != nil {
+		return "", fmt.Errorf("AddCustomer: %v", err)
+	}
+
+	return parsedResponse.AccountNumber, nil
+}
+
+// GetCustomer retrieves a Customer's information for the provided
+// Hex Account Number
+func (svc *CustomerService) GetCustomer(
+	params GetCustomerParams,
+) (*GetCustomer, error) {
+	relURL := fmt.Sprintf("v2/pcc/customers/%s", params.AccountNumber)
+	request, err := svc.Client.BuildRequest("GET", relURL, nil)
+
+	if err != nil {
+		return nil, fmt.Errorf("GetCustomer: %v", err)
+	}
+
+	parsedResponse := &GetCustomer{}
```

Comment:
GetCustomer{} doesn't follow the naming rule. It should be Customer or CustomerResponse.

Reply:
@changgyu-oh good catch. Decided to go with CustomerGetOK to match auto generated code.